appserver/tests/pom.xml

```diff
@@ -42,6 +42,9 @@
 -->
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd"> <modelVersion>4.0.0</modelVersion>
+    <modules>
```

Comment:
The modules for payara samples are in appserver/tests/payara-samples/samples/pom.xml (As you have already done). This should be removed

Reply:
removed!